pkgs/tools/package-management/cargo-update/cargo-lock.patch

```diff
@@ -0,0 +1,739 @@
+diff --git a/Cargo.lock b/Cargo.lock
```

Comment:
Can you upload this to a gist? to avoid increasing the size of nixpkgs. Besides that, LGTM

Reply:
nvm, It's not really blocking for merge